tests/openvino/test_modeling.py

```diff
@@ -1222,24 +1225,24 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "gpt_bigcode": 5,
         "blenderbot": 2,
         "blenderbot-small": 2,
-        "bloom": 5,
+        "bloom": 5 if is_transformers_version(">=", "4.49") else 0,
         "chatglm": 2,
         "codegen": 5,
         "codegen2": 2,
-        "gpt2": 5,
-        "gptj": 5,
-        "gpt_neo": 4,
-        "gpt_neox": 5,
+        "gpt2": 5 if is_transformers_version(">=", "4.42") else 0,
+        "gptj": 5 if is_transformers_version(">=", "4.49") else 0,
+        "gpt_neo": 4 if is_transformers_version(">=", "4.45") else 0,
```

Comment:
why is it changed? I see we have no SDPA for lower transformer version. Before this change we had SPDA op, right?